Jenkinsfile

```diff
@@ -73,7 +73,7 @@ pipeline {
           agent { node { label 'linux' } }
           options { timeout(time: 30, unit: 'MINUTES') }
           steps {
-            mavenBuild("jdk11", "install checkstyle:checkstyle -DskipTests", "maven3", true)
+            mavenBuild("jdk11", "install checkstyle:check -DskipTests", "maven3", true)
```

Comment:
If we turn on error level in checkstyle, wont this will fail the build immediately on the first error?

Reply:
failsOnError is false per default :) http://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html#failsOnError